src/components/panel/panel.scss

```diff
@@ -70,7 +70,6 @@
 :host([loading]) .container,
 :host([disabled]) .container {
   @apply relative;
-  z-index: 1;
```

Comment:
this might break this case:
https://codepen.io/driskull/pen/yLpawmY?editors=1000

Reply:
Good catch. I'll revisit this one.

Reply:
Started looking at this one today and I have a question about the codepen above. What is the use case for setting `position: relative` and `z-index` for the elements in the panel content?

Reply:
Its to illustrate that users might have content with a z-index set and we don't want their content to appear over the panel.

Reply:
This will still be an issue regardless of what we do in the component, right? Users can always modify stacking contexts and set `z-index` to a value higher than we expect ([block example](https://codepen.io/jcfranco/pen/ExobaXe?editors=1000)). 

Sidebar: we could introduce a set of tokens to represent different levels of z-indices used in the design system. This could help our components use the same index depending on the context and better inform users on how to override these values. Some resources:

https://www.lightningdesignsystem.com/design-tokens/#category-z-index
https://designsystem.morningstar.com/legacy/v/2.4.0/visual-language/layering.html
https://www.duetds.com/tokens/#z-index
https://github.com/dojo/widgets#z-index-layering
https://www.smashingmagazine.com/2021/02/css-z-index-large-projects/

Reply:
Based on ☝️, I think we can move forward with this after patches are done. I'll create a separate issue to introduce and use z-index tokens.